Makefile

```diff
@@ -72,4 +72,5 @@ test:
 .PHONY: clean ## Remove all build artifacts
 clean:
 	rm -rf $(BUILD_DIR)
+	rm $(GOPATH)/bin/crc
```

Comment:
I need to remove this from time to time, so putting it in the makefile....

Reply:
:-/